build.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
+# build the docker images for the project
+
+# load the environment variables
+set -o allexport; source .env; set +o allexport;
```

Comment:
hmm, i don't think we should be touching `.env` directly

Reply:
If we use environment variables, it gets quite tricky to also run the batch files directly on the server